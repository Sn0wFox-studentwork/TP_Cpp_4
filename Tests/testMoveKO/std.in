R r1 1 1 0 0
MOVE r2 1 1
EXIT
