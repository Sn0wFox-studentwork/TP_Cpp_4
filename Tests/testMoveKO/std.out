OK
ERR
#r2 is not part of the figure